openapi/tests/openapi_integration/helpers/collection_setup.py

```diff
@@ -11,6 +11,112 @@ def drop_collection(collection_name='test_collection'):
     )
     assert response.ok
 
+def geo_collection_setup(
+        collection_name='test_collection',
+        on_disk_payload=False,
+        on_disk_vectors=None,
+):
+    on_disk_vectors = on_disk_vectors or bool(int(os.getenv('QDRANT__ON_DISK_VECTORS', 0)))
+
+    response = request_with_validation(
+        api='/collections/{collection_name}',
+        method="DELETE",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
+
+    response = request_with_validation(
+        api='/collections/{collection_name}',
+        method="PUT",
+        path_params={'collection_name': collection_name},
+        body={
+            "vectors": {
+                "size": 4,
+                "distance": "Dot",
+                "on_disk": on_disk_vectors
+            },
+            "on_disk_payload": on_disk_payload
+        }
+    )
+    assert response.ok
+
+    response = request_with_validation(
+        api='/collections/{collection_name}',
+        method="GET",
+        path_params={'collection_name': collection_name},
+    )
+    assert response.ok
+
+    response = request_with_validation(
+        api='/collections/{collection_name}/points',
+        method="PUT",
+        path_params={'collection_name': collection_name},
+        query_params={'wait': 'true'},
+        body={
+            "points": [
+                {
+                    "id": 1,
+                    "vector": [0.05, 0.61, 0.76, 0.74],
+                    "payload": {
+                        "value": 1,
+                        "location": {
+                                    "lon": 50.5200,
+                                    "lat": 50.4050
+                                }}
+                },
+                {
+                    "id": 2,
+                    "vector": [0.19, 0.81, 0.75, 0.11],
+                    "payload": {
+                        "value": 2,
+                        "location": {
+                                    "lon": 60.5200,
+                                    "lat": 60.4050
+                                }}
+                },
+                {
+                    "id": 3,
+                    "vector": [0.36, 0.55, 0.47, 0.94],
+                    "payload": {
+                        "value": 3,
+                        "location": {
+                                    "lon": -60.5200,
+                                    "lat": -60.4050
+                                }}
+                },
+                {
+                    "id": 4,
+                    "vector": [0.18, 0.01, 0.85, 0.80],
+                    "payload": {
+                        "value": 4,
+                        "location": {
+                                    "lon": 80.5200,
+                                    "lat": 80.4050
+                                }
+                                }
+                },
```

Comment:
maybe the repo should specify a python formatter, cc @generall

Reply:
sounds good. I added an issue for this https://github.com/qdrant/qdrant/issues/2451